rs/sns/governance/canister/governance_test.did

```diff
@@ -266,7 +266,12 @@ type GetProposalResponse = record {
 
 type GetRunningSnsVersionResponse = record {
   deployed_version : opt Version;
-  pending_version : opt UpgradeInProgress;
+  pending_version : opt record {
```

Comment:
Let's not rename the response API type, instead the input API type can be renamed.

Reply:
Done!